backend/api_types.py

```diff
@@ -1,8 +1,15 @@
 from graphene.relay import Node
-from models import User, Artwork, Portfolio, Achievement
+from models import Settings, User, UserMetrics, Achievement, Artwork, ArtworkMetrics
+from models import Comment, Portfolio, Group
+
 from graphene_mongo import MongoengineObjectType
 
 
+class SettingsType(MongoengineObjectType):
+    class Meta:
+        model = Settings
+        interfaces = (Node,)
```

Comment:
We may not need the `Node` interface here. Node is nice if we are doing pagination or filtering, but it makes the calls more complex because you have to add `edges{node {...}}` to every specification. We may want to make settings a simple query field with no interface so we can call it like this `user{settings {<field>}}` instead of `user { settings { edges { node { <field>}}}}`